nfd/node-feature-discovery/templates/topologyupdater.yaml

```diff
@@ -0,0 +1,156 @@
+{{- if .Values.topologyUpdater.enable -}}
```

Comment:
we want those values to be taken from NFD CR, no? in that case it should be:

{{- if .Values.topologyUpdater -}}